src/liquid/components/ld-modal/ld-modal.css

```diff
@@ -0,0 +1,195 @@
+ld-modal {
+  display: contents;
+}
+
+@keyframes ld-modal-in {
+  from {
+    transform: translateY(2rem);
+  }
+
+  to {
+    transform: translateY(0rem);
+  }
+}
+
+@keyframes ld-modal-out {
+  from {
+    transform: scale(1);
+  }
+
+  to {
+    transform: scale(0.9);
+  }
+}
+
+@keyframes ld-modal-out-mobile {
+  from {
+    transform: translateY(0rem);
+  }
+
+  to {
+    transform: translateY(2rem);
+  }
+}
+
+dialog.ld-modal,
+:host(.ld-modal) dialog {
```

Comment:
Another question: Why don't you simply add `class={cl}` to the `<dialog>` instead of the `<Host>` element? This would probably simplify the selectors.

Reply:
I think I tried this, but ended up reverting to what I have now. If I remember correctly the reason was that the host element needed to have display contents.

Reply:
That's a bit confusing, as :host(.ld-modal) actually IS the host element. (According to your code that says `<Host class={cl}> ...`)

Reply:
Oh, I missed your point! Of course you are right!

Reply:
Changed.